internal/fnruntime/wasmtime.go

```diff
@@ -1,5 +1,4 @@
 //go:build cgo
-// +build cgo
```

Comment:
Why remove this? Is it intentional or just automatically done by the tooling? I seem to remember that the `+build` syntax is deprected.